.pre-commit-config.yaml

```
@@ -0,0 +1,5 @@
+repos:
+  - repo: https://github.com/norwoodj/helm-docs
+    rev: v0.8.0
```

```suggestion
    rev: v0.12.0
```
I just realized that the caasperli example had a version mismatch, sorry.

See https://github.com/hairmare/helm-caasperli/pull/11